hw/ip/aes/pre_sca/alma_post_syn/yosys_synth_template.txt

```diff
@@ -0,0 +1,26 @@
+# This is a custom yosys template that is later patched
```

Comment:
I wrote this by playing around with yosys commands, but of course the template by CocoAlma was a relevant example. I specifically used a similar template from CocoAlma, because it allows me to pass some variables like `{READ_FILES}`, `{TOP_MODULE}` which are populated by `parse.py`. Otherwise, I am afraid one needs to hardcode file names into this script.

Not sure how to go around this licensing issue. Should I maybe implement a python function myself that generates the yosys script (in order to get rid of hardcoding filenames)?

Reply:
We've discussed this offline and you came up with a good solution to handle this. I am resolving the conversation.